sharepoint/sharepoint-ps/sharepoint-online/New-SPOContainerType.md

```diff
@@ -0,0 +1,189 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/new-spocontainertype
+applicable: SharePoint Online
+title: New-SPOContainerType
+schema: 2.0.0
+author: ShreyasSar26
+ms.author: ShreyasSar26
+ms.reviewer:
+---
+
+# New-SPOContainerType
+
+## SYNOPSIS
+
+This cmdlet creates a new container type of standard or trial status. The standard container type can be created with the regular billing structure or direct to customer billing structure.
+
+## SYNTAX
+
+### ParamSet1
+
+```powershell
+New-SPOContainerType –ContainerTypeName <ContainerTypeName> -OwningApplicationId <OwningApplicationId> -AzureSubscriptionId <AzureSubscriptionId> -ResourceGroup <ResourceGroup> -Region <Region>
+```
+
+### ParamSet2
+
+```powershell
+New-SPOContainerType -ContainerTypeName <ContainerTypeName> -OwningApplicationId <OwningApplicationId> -IsPassThroughBilling
+```
+
+### ParamSet3
+```powershell
+New-SPOContainerType –TrialContainerType -ContainerTypeName <ContainerTypeName> -OwningApplicationId <OwningApplicationId>
+```
+
+## DESCRIPTION
+
+This cmdlet creates a new container type of the standard or trial status. A trial container type does not have a billing profile associated with it and has a validity of 30 days. A standard container type has a billing profile associated with it. With the use of -IsPassThroughBilling, we can create a direct to customer billed container type.
```

Comment:
>-IsPassThroughBilling [](http://example.com/codeflow?start=263&length=21)

add backticks around it, i.e.

```
`-IsPassThroughBilling`
```

Reply:
Closed

Reply:
@ShreyasSar26 should be backtick ` not regular '

Reply:
Changed to backtick. Closed